themes/bootstrap3/js/common.js

```diff
@@ -240,19 +240,31 @@ var VuFind = (function VuFind() {
   };
 
   var loadHtml = function loadHtml(_element, url, data, success) {
-    var $elem = $(_element);
-    if ($elem.length === 0) {
+    var element = document.querySelector(_element);
```

Comment:
One question: I notice that _element is likely to be a jQuery object in some situations. Does passing a jQuery object to querySelector work reliably, or should we perhaps either change the calling code or de-jQuery the object here?

In any case, I expect that #3283 is going to be merged very soon, which will greatly reduce the number of calls to the function by eliminating a large number of redundant template files. If it's worth changing the calls as well as the function, that should be much easier very soon...

Reply:
As noted earlier, #3283 has been merged, so it may be worth merging the latest dev branch into this, especially if it's necessary to adjust the calls to the modified function.

Reply:
Will do it today @demiankatz